scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
```

Comment:
Isn't this going to be a consistent look throughout shoreditch theme?
![3_1b_contact_tab_compressed](https://cloud.githubusercontent.com/assets/5058867/25324121/8f2f34ce-28e1-11e7-94a3-7e958bc5acf2.png)

Also @dhavalsharma added margin for the `btn-group`, not for this.